app/src/main/res/layout/question_player_fragment.xml

```diff
@@ -68,9 +68,9 @@
         style="@style/Widget.AppCompat.ProgressBar.Horizontal"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
```

Comment:
I think it would look nice, if we keep it exact 12dp height as per mocks rather than wrap_content.

Reply:
Okay,  Done